uiplib/gui/mainGui.py

```diff
@@ -80,6 +80,8 @@ def next_wallpaper(self):
     def prev_wallpaper(self):
         """Preview previous wallpaper."""
         self.index -= 1
+        if self.index < 0:
+            self.index = len(self.images) - 1
```

Comment:
maybe make it `self.index = self.index - 1 if self.index != 0 else len(self.index) -1`